protoc-gen-gqlgencfg/main.go

```diff
@@ -70,21 +85,38 @@ func (p *plugin) Generate(file *generator.FileDescriptor) {
 
 			cfg := &config{
 				SchemaFilename: []string{},
-				Exec:           packageConfig{Filename: "exec.go"},
-				Model:          packageConfig{Filename: "models.go"},
+				Exec:           packageConfig{Filename: path.Join(p.Gendir, "generated.gen.go")},
+				Model:          packageConfig{Filename: path.Join(p.Gendir, "models.gen.go")},
+				Resolver:       packageConfig{Filename: path.Join(p.Gendir,"resolver.go"), Type: "resolver"},
```

Comment:
In many cases we don't need resolvers because `protoc-gen-gogqlgen` already resolves endpoints, so I think it is better to let the user decide if he wants resolvers, this means adding a option